downstream/modules/platform/proc-controller-add-users-job-templates.adoc

```diff
@@ -4,13 +4,20 @@
 
 .Procedure
 
-. From the navigation panel, select {MenuAMCredentials}.
+. From the navigation panel, select {MenuAECredentials}.
 . Select the credential that you want to assign to additional users.
-. Click the *Access* tab.
+. Click the *User Access* tab.
 You can see users and teams associated with this credential and their roles.
-. Choose a user and click btn:[Add].
 If no users exist, add them from the *Users* menu.
-For more information, see xref:assembly-controller-users[Users].
-. Select *Job Templates* to display the job templates associated with this credential, and which jobs have run recently by using this credential.
+For more information, see TBD[Users].
+. Select the user(s) that you want to give access to the credential and click btn:[Next].
+. From the *Select roles to apply* page, select the roles you want to add to the User.
+. Click btn:[Next].
+. A window is displayed stating whether the action was succesful. 
+. Click btn:[Close]. 
+//What happens if it fails?
+//There's a Finish button on this page but it doesn't seem to be necessry if you click close.
+. The *User Access* page displays the summary information.
+//Select *Job Templates* to display the job templates associated with this credential, and which jobs have run recently by using this credential.
```

Comment:
Since the user is dropped back onto the *User Access* page, I suggest including the selection of the *Job Templates* tab. Something like:
. Select the *Job Templates* tab to choose a job template to which you want to assign this credential.
. Click btn[Add].